diff --git i/CMakeLists.txt w/CMakeLists.txt
index dd602fdacf..bbed13d2fb 100644
--- i/CMakeLists.txt
+++ w/CMakeLists.txt
@@ -1186,8 +1186,7 @@ endif()
 
 option(WITH_JNI "build with JNI" OFF)
 # Tests are excluded from Release builds
-CMAKE_DEPENDENT_OPTION(WITH_TESTS "build with tests" ON
-  "CMAKE_BUILD_TYPE STREQUAL Debug" OFF)
+option(WITH_TESTS "build with tests" ON)
 option(WITH_BENCHMARK_TOOLS "build with benchmarks" ON)
 option(WITH_CORE_TOOLS "build with ldb and sst_dump" ON)
 option(WITH_TOOLS "build with tools" ON)
