diff -ruN a/jack-dssi-host/jack-dssi-host.c b/jack-dssi-host/jack-dssi-host.c
--- a/jack-dssi-host/jack-dssi-host.c	2011-05-24 06:21:17.000000000 +0200
+++ b/jack-dssi-host/jack-dssi-host.c	2024-08-22 22:04:45.479824221 +0200
@@ -119,7 +119,7 @@
 void osc_error(int num, const char *m, const char *path);
 
 int osc_message_handler(const char *path, const char *types, lo_arg **argv, int
-		      argc, void *data, void *user_data) ;
+		      argc, lo_message data, void *user_data) ;
 int osc_debug_handler(const char *path, const char *types, lo_arg **argv, int
 		      argc, void *data, void *user_data) ;
 
@@ -1919,7 +1919,7 @@
 }
 
 int osc_message_handler(const char *path, const char *types, lo_arg **argv,
-                        int argc, void *data, void *user_data)
+                        int argc, lo_message data, void *user_data)
 {
     int i;
     d3h_instance_t *instance = NULL;
