--- a/PKGBUILD
+++ b/PKGBUILD
@@ -16,13 +16,13 @@
 license=(Zlib)
 url="https://www.zlib.net/"
 source=(https://github.com/madler/zlib/releases/download/v$pkgver/$pkgname-$pkgver.tar.xz{,.asc}
-        https://github.com/madler/zlib/commit/36ff1be48ef696cc67b0855f7c8537ce0276210d.patch)
+        36ff1be48ef696cc67b0855f7c8537ce0276210d.patch)
 sha512sums=('cf3d49fbabddc57cca99858feeca8f910e9de42a16014cddd406814d2d24ee33fee2af3805d7efbb1b04b05f55818092b000daf82502b675df65f2512c353f73'
             'SKIP'
-            '39a4ad7f473e4c170fb91a2e14860eaa6e728e9b710185bf9dbb23b949064fe7e51e36a15ec5ea110760ca8af15f8d2450e02e1a442af66516fe3e8b2debda0d')
+            'ef6d5df8e4710454b8f929d46b8587997cd70d14295d20182af8f55a4129c288a0498fcc2dc5a88e1ffd944a9dceb1b375b2926f10b8dacf142f46bdec558e4f')
 b2sums=('fd3a6e4c275a925f4814de487075c136e28421ea34ca7f93141781e7a414b392cd849fbf96cfcb24e825441df5f3c64be54276c0f617dc2d5c865d4ab260e83c'
         'SKIP'
-        '489546e224668e177e1358c7bb56bf9206a292d78cdb5fbf437533d07a4dffc60984499abd17fc3c776a09da78caacc9c9b35bf20194b2d4fe8c9224d18da5d9')
+        'e2e21cbce057a57df1e7b2196dfe3971799371c44cb40f12d28027a32d75ec67961f338b54dad29d83c4d3fd4481b30bc8d07544aacb2c218471d635e243261c')
 validpgpkeys=('5ED46A6721D365587791E2AA783FCD8E58BCAFBA')  # Mark Adler <madler@alumni.caltech.edu>
 
 prepare() {
--- /dev/null
+++ b/36ff1be48ef696cc67b0855f7c8537ce0276210d.patch
@@ -0,0 +1,22 @@
+From 36ff1be48ef696cc67b0855f7c8537ce0276210d Mon Sep 17 00:00:00 2001
+From: Rui Chen <rui@chenrui.dev>
+Date: Tue, 17 Feb 2026 10:48:39 -0500
+Subject: [PATCH] contrib/minizip: install ints.h
+
+Signed-off-by: Rui Chen <rui@chenrui.dev>
+---
+ contrib/minizip/Makefile.am | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/contrib/minizip/Makefile.am b/contrib/minizip/Makefile.am
+index d343011ebc..b7dea4f7e7 100644
+--- a/contrib/minizip/Makefile.am
++++ b/contrib/minizip/Makefile.am
+@@ -27,6 +27,7 @@ libminizip_la_LDFLAGS = $(AM_LDFLAGS) -version-info 1:0:0 -lz
+ minizip_includedir = $(includedir)/minizip
+ minizip_include_HEADERS = \
+ 	crypt.h \
++	ints.h \
+ 	ioapi.h \
+ 	mztools.h \
+ 	unzip.h \
